### ISSUE TITLE
Dialog Tasks

### ISSUE THREAD

**Issue by yizhongw** [allenai/natural-instructions#738]
It looks like we don't have many dialog tasks, especially for response generation. 

This [webpage](https://github.com/sebastianruder/NLP-progress/blob/master/english/dialogue.md) contains a list of dialog tasks we can consider. I think we should include at least some of the popular tasks. 

My suggestion list:
Dialog act classification - Switchboard corpus
Dialog state tracking - Wizard-of-Oz, MultiWOZ
Dialog generation - PERSONA-CHAT, [DailyDilog++](https://iitmnlp.github.io/DailyDialog-plusplus/), [DSTC7-challenge](https://github.com/mgalley/DSTC7-End-to-End-Conversation-Modeling).

**Comment by Palipoor** [comment]
For MultiWOZ we have these:
```

task638_multi_woz_classification
task639_multi_woz_user_utterance_generation
task1394_meta_woz_task_classification
```

**Comment by yizhongw** [comment]
> For MultiWOZ we have these:
> 
> ```
> 
> task638_multi_woz_classification
> task639_multi_woz_user_utterance_generation
> task1394_meta_woz_task_classification
> ```

Right. I also realized we do have more dialogue tasks than I previously thought. But let's still add these common datasets.
